CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+This project is intended to provide categorized links to other resources in a
+format that's easy to navigate. The information it contains is currently
+divided into four sections, each with its own guidelines for contributions as
+detailed below.
+
+To submit a contribution, simply [fork this repository](https://help.github.com/articles/fork-a-repo/)
+and [submit a pull request](https://help.github.com/articles/creating-a-pull-request/).
+
+## Concepts
+
+Additions to this section should be rare. Each subsection references a single
+high-level concept related to asynchronous programming and contains links
+specific to that concept, generally introductory tutorials or references.
+
+## Projects
+
+Projects are collections of related libraries, as opposed to a single library
+or framework, and generally have an independent web site dedicated to them.
+Individual libraries and frameworks should be listed in the next section.
+
+## Repositories
+
+This section lists source control repositories for individual libraries and
+frameworks that are in some way related to asynchronous programming in PHP,
+loosely categorized into subsections based on purpose.
+
+A repository listed in this section may be part of a project listed in the
+Projects section, but this is not necessary for its inclusion.
+
+To keep the noise-to-signal ratio low, no repository may be listed more than
```

Comment:
I believe there are times, that one module can contain logic applicable to more than one repository. For example right now there are separate sections for "messaging" and "sockets". It is highly possible for two libraries to fulfill both repositires. In future, there might be even more overlapping. I would suggest to change "no repository may be listed more than once" to "no repository may be listed more than once, if there is no strong reason to".